security-release-process.md

```diff
@@ -157,7 +157,30 @@ These steps should be completed within the 1-7 days of Disclosure.
 
 - The Fix Lead and the Fix Team will create a [CVSS](https://www.first.org/cvss/specification-document) score using the [CVSS Calculator](https://www.first.org/cvss/calculator/3.0). They will also use the [Severity Thresholds - How We Do Vulnerability Scoring](#severity-thresholds---how-we-do-vulnerability-scoring) to determine the effect and severity of the bug. The Fix Lead makes the final call on the calculated risk; it is better to move quickly than make the perfect assessment.
 - The Fix Lead will [Assign a CVE ID to the vulnerability from the CVE Numbering Authority](/cna-handbook.md#assign-a-cve-id-to-the-vulnerability).
-- The Fix Team will notify the Fix Lead that work on the fix branch is complete once there are LGTMs on all commits in the private repo from one or more relevant assignees in the relevant OWNERS file.
+- The Fix Lead will sync the master branch and active release branches from the public repo to the private security repo in the kubernetes-security GitHub org.
+- The Fix Team will notify the Fix Lead that work on the master and release branches is complete once there are LGTMs on all commits in the private repo from one or more relevant assignees in the relevant OWNERS file.
+- These pull requests should not be merged on the private kubernetes-security GitHub repo. The Fix Lead should reference these PRs for embargoed patch development and cherry-picking to the public repo on the fix release day. 
+
+**Sync kubernetes-security from public repositories**
```

Comment:
Two thoughts:

- Can we turn this into a shell script that we can have in the repo?
- Maybe it makes more sense to have this automated via either a GitHub action or whatever the release team uses on k/k?

Reply:
you can add it under https://github.com/kubernetes/committee-security-response/tree/main/tools also can you add some text about using https://github.com/kubernetes/committee-security-response/blob/main/tools/patchformat.py for patches to send out for embargo email?

Reply:
Shell script to sync branches has been added to the PR. I will push a follow-up for how to use patchformat.py when I go through those steps for CVE-2025-5187 (Sergey is working on some last changes to the patch files).